src/util.c

```diff
@@ -0,0 +1,6 @@
+#include "util.h"
+
+bool menu_layer_menu_index_selected(MenuLayer *menu_layer, MenuIndex *index) {
```

Comment:
There is an API now called menu_layer_is_index_selected, that you can replace this method with